dracut/30afterburn/azure-hostname.service

```diff
@@ -0,0 +1,15 @@
+# Azure needs to fetch hostname from metadata
+# during the initramfs
+
+[Unit]
+Description=Afterburn hostname
```

Comment:
Super minor: let's do descriptions in Title Case?

Reply:
Will do